include/pfs/types.hpp

```diff
@@ -604,6 +604,22 @@ struct id_map
     size_t length = UINT32_MAX;
 };
 
+// Based on https://elixir.bootlin.com/linux/v5.19.10/source/net/ipv4/fib_trie.c#L2998
```

Comment:
Since line numbers and implementations change, and you don't always want to see a specific linux kernel version, maybe it's better to be consistent with other hints in the file and say:
`// Hint: See 'fib_route_seq_show @ fib_trie.c'`
This exact same function existed since 2.6.32 (https://elixir.bootlin.com/linux/v2.6.32/source/net/ipv4/fib_trie.c#L2567)